### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,13 +71,13 @@ TO DRIVES THAT ARE VENDOR SAMPLES OR NOT PUBLICLY AVAILABLE.
 
 ## II. Compilation and installation
 
-### II.1. Requiremensts
+### II.1. Requirements
 
-libzbc requires  that the  autoconf, automake and  libtool development
+libzbc requires that the autoconf, automake and libtool development
```
Comment (damien-lemoal) [inline README.md]: The extra spaces are there to align line length... Keep them please.

Comment (harshavardhana) [reply README.md]: that's pretty much anti-markdown @damien-lemoal

Comment (damien-lemoal) [reply README.md]: Yes and no. Rendering of the file in a web browser will ignore the spaces, but readability is in my opinion better when the file is viewed in a terminal.

Comment (harshavardhana) [reply README.md]: Resolved

**Comment by damien-lemoal** [comment]
Merged. Thanks !

**Pull request merged**